src/components/dev-dot-content/mdx-components/mdx-heading-permalink/mdx-heading-permalink.module.css

```diff
@@ -24,7 +24,7 @@
 		}
 	}
 
-	&:focus {
+	&:focus-visible {
```

Comment:
Switched to `:focus-visible` here to avoid a lingering focus style when clicking the permalink link icon.